phpstan-baseline.neon

```diff
@@ -1240,46 +1240,6 @@ parameters:
 			count: 1
 			path: Tests/Docker/Runner/DataLoader/NullDataLoaderTest.php
 
-		-
-			message: "#^Cannot access offset 'authorization' on mixed\\.$#"
-			count: 3
-			path: Tests/Runner/ConfigFileTest.php
-
-		-
-			message: "#^Cannot access offset 'data_column' on mixed\\.$#"
-			count: 1
-			path: Tests/Runner/ConfigFileTest.php
-
-		-
-			message: "#^Cannot access offset 'parameters' on mixed\\.$#"
-			count: 5
-			path: Tests/Runner/ConfigFileTest.php
-
-		-
-			message: "#^Cannot access offset 'primary_key_column' on mixed\\.$#"
-			count: 1
-			path: Tests/Runner/ConfigFileTest.php
-
-		-
-			message: "#^Cannot access offset 'string_length' on mixed\\.$#"
-			count: 1
-			path: Tests/Runner/ConfigFileTest.php
-
-		-
-			message: "#^Parameter \\#1 \\$json of function json_decode expects string, string\\|false given\\.$#"
-			count: 3
-			path: Tests/Runner/ConfigFileTest.php
-
-		-
-			message: "#^Parameter \\#2 \\$array of static method PHPUnit\\\\Framework\\\\Assert\\:\\:assertArrayHasKey\\(\\) expects array\\|ArrayAccess, mixed given\\.$#"
-			count: 3
-			path: Tests/Runner/ConfigFileTest.php
-
-		-
-			message: "#^Parameter \\#2 \\$array of static method PHPUnit\\\\Framework\\\\Assert\\:\\:assertArrayNotHasKey\\(\\) expects array\\|ArrayAccess, mixed given\\.$#"
-			count: 3
-			path: Tests/Runner/ConfigFileTest.php
-
```

Comment:
Znejakyho duvodu mi zacal padat phpstan na `ConfigFileTest` ze jedna z ignorovanych chyb se projevila menekrat nez je ocekavano- Tak jsem to rovnou vycistil.